README.md

```diff
@@ -1 +1,34 @@
-# Lucas-Comamala-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Project
+
+GitHub repository for my personal portfolio
+
+## Built With
+
+- HTML
+- CSS
+- Figma template
+
```

Comment:
- [ ] Your readme is missing important sections. Please add the missing sections from [Microverses readme template](https://github.com/microverseinc/readme-template). And please don't forget to customize its content to the details of your project.